roles/pulp_user/tasks/main.yml

```diff
@@ -63,46 +95,33 @@
       is_staff: "{{ item.is_staff | default(None) }}"
       is_active: "{{ item.is_active | default(None) }}"
     body_format: form-urlencoded
-  loop: "{{ pulp_users }}"
+  loop: "{{ update_users }}"
   loop_control:
     label: "{{ item.username }}"
   register: result
-  when:
-    - item.username in usernames
-    - item.state is not defined or item.state != 'absent'
-  changed_when:
-  # The pulp API currently does not report when a change is made, so we must
-  # manually check
-    - result.json not in users_list_result.json.results
-    - result.status == 200
+  changed_when: true
```

Comment:
I guess we lose change tracking, assuming it worked correctly before

Reply:
Yeah, you don't get the tracking inside the loops anymore, which is a bit of a shame, though you do get Create/Remove/Update **any** user/group